testing/fstest/fstest_main.c

```diff
@@ -1183,6 +1191,11 @@ int main(int argc, FAR char *argv[])
   free(ctx->files);
   free(ctx);
 
+  printf("File system tests done... OK: %d, FAILED: %d\n", tests_ok,
+                                                           tests_err);
+
+  tests_ok = tests_err = 0;
```

Comment:
move to the start of main, since the program may exit in the middle of main

Reply:
Done